packages/suite/src/components/suite/FormattedCryptoAmount.tsx

```diff
@@ -57,21 +66,25 @@ export const FormattedCryptoAmount = ({
 
     const isSatoshis = areSatsSupported && areSatsDisplayed;
 
+    // convert to satohsis if needed
     if (isSatoshis) {
-        formattedValue = formatCurrencyAmount(
-            Number(networkAmountToSatoshi(String(value), symbol as NetworkSymbol)),
-        ) as string;
+        formattedValue = networkAmountToSatoshi(String(value), symbol as NetworkSymbol);
 
         formattedSymbol = isTestnet ? `sat ${symbol?.toUpperCase()}` : 'sat';
     }
 
+    // format truncation + locale (used for balances) or just locale
+    if (isBalance) {
+        formattedValue = formatCoinBalance(String(formattedValue), locale);
+    } else {
+        formattedValue = formatCurrencyAmount(Number(formattedValue), locale);
```

Comment:
```suggestion
        formattedValue = formatNumber(Number(formattedValue), locale);
```

Currency amount in formatted crypto amount does not make sense

Reply:
97bf0112f9b74c51f4c29d88982cf9e0e3388d9c